.github/workflows/build_with_pinned_versions.yml

```diff
@@ -0,0 +1,41 @@
+name: JEDI Bundle Pinned Version Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  buildbundle:
+    name: Run jedi_bundle with pinned_versions
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.12"]
```

Comment:
Any particular reason to skip 3.11? (If it works on 3.12 and 3.10, presumably we're fine; just curious).

Reply:
No particular reason, just thought I'd limit the matrix to three versions but I can add in 3.11